c/anagrams_kata/anagrams_finder.c

```diff
@@ -0,0 +1,27 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include <anagrams_finder.h>
+
+// anagrams_finder_find finds anagrams matching the input word
+static void anagrams_finder_find(struct anagrams_finder_s *_this, char *word)
+{
+    printf("%s not implemented!\n", __func__);
+}
+
+// anagrams_finder_new returns an instance of an anagrams_finder_t based on
+// a wordlist file.
+anagrams_finder_t *anagrams_finder_new(char *wordlist)
+{
+    anagrams_finder_t *anagrams_finder_ref = malloc(sizeof(anagrams_finder_t));
+
+    anagrams_finder_ref->wordlist = wordlist;
```

Comment:
The other implementations load up the file when creating the finder.
So far my impression was that the kata is not about this part. Would require reading a FILE then.